crates/lib/src/deploy.rs

```diff
@@ -467,14 +462,10 @@ pub(crate) async fn prepare_for_pull_unified(
     // Configure the importer to use bootc storage as an additional image store
     let mut config = ostree_ext::containers_image_proxy::ImageProxyConfig::default();
     let mut cmd = Command::new("skopeo");
-    // Use the actual physical path to bootc storage
-    // During install, this is the target disk's mount point; otherwise default to /sysroot
-    let sysroot_base = sysroot_path
-        .map(|p| p.to_string())
-        .unwrap_or_else(|| "/sysroot".to_string());
+    // Use the physical path to bootc storage from the Storage struct
```

Comment:
This is OK but I've been trying to ensure that bootc uses file descriptors and not file paths as much as possible. The problem here is the case where we have a fd but need to pass that state down to a subprocess which expects a file path.

podstorage.rs goes to quite some effort to do this for the primary storage, see `new_podman_cmd_in` - I think we could use a similar technique here.